source/tutorial/upgrade-config-servers-to-replica-set.txt

```diff
@@ -25,6 +25,14 @@ The following procedure upgrades three mirrored config servers to a
 procedure, all the sharded cluter binaries must be at least version
 3.2.4.
 
+Please note that during this procedure there will be a brief window where 
+the config servers will be read-only.  This means that certain catalog
```

Comment:
s/This means that certain catalog operations will not be possible during this window/During this period, certain catalog operations will fail if attempted./